### PR TITLE
contribution flow added to documentation 

### PR DESCRIPTION
**Pull request by Jiggy9**
I added necessary contribution flow to documentation. Let me know if something is missing or need to modify.

### CODE REVIEW

**Review by AshutoshDash1999 — COMMENTED** [review]
@Jiggy9 please add screenshots

File: README.md [hunk 1]
```diff
@@ -40,6 +40,40 @@ b. To use the icons, click on your preferred icons, and paste inside your body t
...
+      ```
+      git push -u origin your-branch-name
+      ```
+      replacing your-branch-name with the name of the branch you created earlier.
+      
+7. Submit your changes for review 
+    If you go to your repository on GitHub, you'll see a Compare & pull request button. Click on that button. Now add appropriate title and comment on it and submit
+    the pull request.
+
 ## Contributors
```
Comment (AshutoshDash1999) [inline README.md]: please don't add this part now as this is the first time we are participating in open source

Comment (Jiggy9) [reply README.md]: @AshutoshDash1999 Sure I'll add screenshots of each steps to improve it, but before let me know which part I should not add can you clarify more?

Comment (AshutoshDash1999) [reply README.md]: The contributors part

Comment (Jiggy9) [reply README.md]: Okay I didn't added that It was already there before, well If u think it's not necessary I'll remove it and I'll add screenshots and will send PR ASAP.. thank you for clarify.

Comment (AshutoshDash1999) [reply README.md]: hey @Jiggy9, sorry my bad, I thought the contributors part that you are adding is blank. So I meant not to set it blank. You can go forward with it. Just add screenshots of every step. That will be great.

**Comment by AshutoshDash1999** [comment]
@Jiggy9 can you please share a screenshot of whole readme?

**Comment by Jiggy9** [comment]
Screenshot_2023-05-21-12-16-42-01

**Comment by AshutoshDash1999** [comment]
seems good to me. will review it and accept the PR

**Review by ramanujamgond — APPROVED** [review]
Thank you for your contribution

**Pull request merged**